### ISSUE TITLE
Higher geographic resolution

### ISSUE THREAD

**Issue by MHWauben** [unterbrink/2020trends#1]
Hi Michelle, I found this analysis - very creative! 
I am looking to do something similar, but at a higher geographic resolution: are there geo-codes for geographies smaller than 'GB-ENG'? The Google Trends page itself lets you analyse searches by major cities, are those available in the API?

**Comment by unterbrink** [comment]
Absolutely--I used GB-ENG, but there are actually > 450 possible geo codes in the UK (e.g., LON, MAN, DUB). If you want to get a list of the options in R:

```
library(gtrendsR)
geo.codes = sort(unique(countries[substr(countries$sub_code, 1, 2) == "GB", ]$sub_code))
geo.codes
```
More details here: https://www.displayr.com/extracting-google-trends-data-in-r/

**Comment by MHWauben** [comment]
Thank you! Very helpful. However, using any of the non-country ones (GB-ENG works, but GB-ZET doesn't) I get the following error: 

`Error in get_widget(comparison_item, category, gprop, hl, cookie_url,  : widget$status_code == 200 is not TRUE`

Note: I only ran 5 queries, so it's not Google throttling me because of too many requests.

**Comment by unterbrink** [comment]
I see what you mean. This seems to work for some US areas (e,g, "US-AZ", but not for GB locations). Additionally, when you look at trends area you can see results over time and by city, but not by city over time:

`gtrends(c("apple"), geo = c("GB"), time="2019-01-01 2019-12-31")`

If you could get city x time, the analysis would work. Going to reopen over at the gtrendsR repo (https://github.com/PMassicotte/gtrendsR/issues/204)

**Comment by JBleher** [comment]
> Absolutely--I used GB-ENG, but there are actually > 450 possible geo codes in the UK (e.g., LON, MAN, DUB). If you want to get a list of the options in R

I think this is wrong. There are not 400 possible geo codes. Have a look at the google.com/trends homepage. There are only 4 regions for the United Kingdom for which you can actually download time series. The grendsR data set that contains geo codes that come from another homepage, not from google. See: https://github.com/PMassicotte/gtrendsR/issues/324
